packages/storybook/stories/MessageThread/placeholdermessages.ts

```diff
@@ -70,6 +71,30 @@ export const GenerateMockNewChatMessageFromOthers = (): ChatMessage => {
   };
 };
 
+export const GenerateMockNewChatMessageWithInlineImage = (): ChatMessage => {
+  return {
+    messageType: 'chat',
+    ...UserThree,
+    messageId: Math.random().toString(),
+    content: 'Check out this image I just sent!',
+    createdOn: new Date('2020-04-13T00:00:00.000+07:01'),
+    mine: false,
+    attached: false,
+    contentType: 'text',
```

Comment:
should this be something else once there is a image to put here? like multimedia or something? (not too sure the SDK typing for this.)

Reply:
Yes the ChatSDK will return it as a `html` type. Maybe I can just update this one to more accurately represent what the service will send.

Reply:
Updated

Reply:
I noticed it accepts the type of 'html' or 'richtext/html' do you know what the differences are? based on my testing chat sdk only sends 'html'